openqa-qam-sle

```diff
@@ -0,0 +1,220 @@
+dict_group["15-SP6"]="546"
+dict_group["12-SP3"]="106"
+dict_group["12-SP5"]="282"
+dict_group["15-SP4-TERADATA"]="521"
+dict_group["12-SP3-TERADATA"]="191"
+
+_ok() {
+    echo -e "\033[01;32m$1\033[0m"
+}
+
+_ko() {
+    echo -e "\033[01;31m$1\033[0m"
+}
+
+_warn() {
+    echo -e "\033[01;33m$1\033[0m"
+}
+
+_title() {
+    echo -e "\033[01;36m$1\033[0m"
+}
+
+search_maintenance_single_incidents() {
+    _title "Maintenance: Single Incidents / Core Incidents"
+    local ReviewRequestID=$1
+
+    local ID
+    ID="$(echo "$ReviewRequestID" | awk -F':' '{printf "%s", $3}')" # The ID of the update
+    # local RR="$(echo "$ReviewRequestID" | awk -F':' '{printf "%s", $4}')" # The Release Request of the update
+
+    local incident_settings
+    incident_settings=$(curl -sk -X GET "http://dashboard.qam.suse.de/api/incident_settings/$ID")
+
+    local build
+    build=$(echo "$incident_settings" | jq -r '.[0].settings.BUILD')
+
+    local versions
+    versions=$(echo "$incident_settings" | jq -r '.[].settings.VERSION' | sort | uniq)
+
+    local versions_teradata
+    versions_teradata=$(echo "$incident_settings" | jq -r '[.[]| { KEY:(.settings.VERSION + "|" +.settings.FLAVOR)}] | .[].KEY' | sort | uniq | grep TERADATA | sed -e 's/|.*-TERADATA$/-TERADATA/' | sort | uniq)
+
+    for version in $versions $versions_teradata; do
+        # echo "# Version: $version"
+        local groupid=${dict_group[$version]}
+        [[ -z "$groupid" ]] && continue
+
+        local count_url="https://openqa.suse.de/api/v1/jobs/overview?distri=sle&version=${version%%-TERADATA}&build=${build}&groupid=${groupid}"
+        local count
+        count=$(curl -sk -X GET "$count_url" | jq -r '. | length')
+        [[ $count -eq 0 ]] && continue
+
+        printf 'Version: "%s" Build: "%s"\n' "$version" "$build"
+        version=${version%%-TERADATA}
+        printf 'https://openqa.suse.de/tests/overview?distri=sle&version=%s&build=%s&groupid=%s\n' "$version" "$build" "$groupid"
+
+        local running_url="https://openqa.suse.de/api/v1/jobs/overview?distri=sle&version=${version}&build=${build}&groupid=${groupid}&state=scheduled&state=running"
+        local running
+        running=$(curl -sk -X GET "$running_url" | jq -r '. | length')
+        [[ $running -gt 0 ]] && {
+            _warn "RUNNING / SCHEDULED ($running jobs) Awaiting completion..."
+            echo ""
+            continue
+        }
+
+        local failed_url="https://openqa.suse.de/api/v1/jobs/overview?distri=sle&version=${version}&build=${build}&groupid=${groupid}&result=failed&result=incomplete&result=timeout_exceeded"
+        local failed
+        failed=$(curl -sk -X GET "$failed_url" | jq -r '. | length')
+        [[ $failed -eq 0 ]] && _ok "PASSED"
+        [[ $failed -gt 0 ]] && _ko "FAILED ($failed jobs)"
+        echo ""
+    done
+    echo "---"
+}
+
+search_maintenance_aggregated() {
+    _title "Maintenance: Aggregated updates / Core Maintenance Updates"
+    local ReviewRequestID=$1
```

Comment:
I'd stick to snake case in bash scripts, especially because your other variables are also using snake case already:
```suggestion
    local review_request_id=$1
```